src/main/java/tatracker/ui/HelpWindow.java

```diff
@@ -34,16 +35,21 @@
      *
      * @param root Stage to use as the root of the HelpWindow.
      */
-    public HelpWindow(Stage root) {
+    public HelpWindow(Stage root, Logic logic) {
```

Comment:
Seem like you need the GuiSettings not the Logic class. Giving access to the logic here is dangerous.